radio/src/gui/480x272/theme.h

```diff
@@ -26,7 +26,7 @@ class BitmapBuffer;
 #define MAX_THEME_OPTIONS              5
 
 class Theme;
-void registerTheme(Theme * theme);
+extern void registerTheme(Theme * theme);
```

Comment:
`extern` is not needed in my opinion. There are several other places with the same issue. http://stackoverflow.com/questions/11712707/extern-functions-in-c-vs-c